app/controllers/plans_controller.rb

```diff
@@ -253,7 +241,7 @@ def update
         #@plan.save
         if @plan.update(attrs) #_attributes(attrs)
           format.html do
-            redirect_to overview_plan_path(@plan),
+            redirect_to plan_contributors_path(@plan),
```

Comment:
I like the idea of routing them to the contributors page after updating the plan